batch7crud-roll7/src/main/webapp/static/css/layout.css

```diff
@@ -0,0 +1,158 @@
+.header {
+    width: 100%;
+    background: #3e9cca;
+}
+
+.header-text {
+    margin: 0 auto;
+    padding-top: 50px;
+    padding-bottom: 50px;
+    color: #fff;
+    text-align: center;
+    font-size: 32px;
+}
+
+.mainbody {
+    width: 100%;
+    background: #eee;
+    height: 500px;
+}
+
+.login-container {
+    width: 300px;
+    margin: 0px auto;
+    padding: 110px 10px;
+    background: #eee;
+    position: relative;
+}
+
+.login-container input[type="text"] {
+    width: 90%;
+    display: block;
+    height: 32px;
+    border: 1px solid #4D90FE;
+    padding: 5px;
+    border-radius: 5px;
+    margin: 10px auto;
+}
+
+.login-container input[type="password"] {
+    width: 90%;
+    display: block;
+    height: 32px;
+    border: 1px solid #4D90FE;
+    padding: 5px;
+    border-radius: 5px;
+    margin: 10px auto;
+}
+
+.login-container input[type="number"] {
+    width: 90%;
+    display: block;
+    height: 32px;
+    border: 1px solid #4D90FE;
+    padding: 5px;
+    border-radius: 5px;
+    margin: 10px auto;
+}
+
+.error{
+    color: red;
+}
+
+.login-container input[type="submit"] {
+    width: 90%;
+    display: block;
+    background: #4D90FE;
+    padding: 5px;
+    border-radius: 5px;
+    margin: 0 auto;
+    color: #fff;
+    font-size: 16px;
+    line-height: 32px

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a semicolon at the end of this declaration [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Asemicolon-declaration)